### PR TITLE
Doc file changes and updates

### PR DESCRIPTION
**Pull request by mcherem**
Deleting files that are now combined, adding a certificate file, doc edits.
This is the final review for Dev.

### CODE REVIEW

File: modules/ROOT/pages/_partials/endpoint-sftp-receive-target.adoc [hunk 1]
```diff
@@ -34,10 +34,15 @@ Required: Yes +
 | Password for the FTP Server. Required if the server is authenticated. +
 Required: Yes +
 
-|Source Path
+|Source Path - receive only
```
Comment (manikmagar) [inline modules/ROOT/pages/_partials/endpoint-sftp-receive-target.adoc:37]: Should we use similar display style for `Archival Path` in SFTP?

File: modules/ROOT/pages/activity-tracking.adoc [hunk 2]
```diff
@@ -33,11 +33,12 @@ Once you have created a message flow, you can begin to track its activity.
 . Select *Activity* from the left navigation.
 
 . Locate a transmission you transmission you are interested in.
```
Comment (manikmagar) [inline modules/ROOT/pages/activity-tracking.adoc]: `transmission you` is repeated.

Comment (mcherem) [reply modules/ROOT/pages/activity-tracking.adoc]: Fixed.

File: modules/ROOT/pages/_partials/endpoint-sftp-receive-target.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/_partials/endpoint-sftp-receive-target.adoc:37]: Yes, fixed.

**Review by mcherem — COMMENTED** [review]
Entered all of Dev's changes to date.

**Review by amitfernandes — APPROVED** [review]
Approved

**Pull request merged**